recipe/meta.yaml

```diff
@@ -61,30 +61,30 @@ source:
       - patches/0006-Win32-Do-not-download-externals.patch
       - patches/0007-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
       # https://github.com/python/cpython/pull/28397
+      - patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch
       # Patch doesn't apply, but is Debian-specific anyway.
       # - patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch
       - patches/0009-Disable-registry-lookup-unless-CONDA_PY_ALLOW_REG_PA.patch
       - patches/0010-Unvendor-openssl.patch
       - patches/0011-Unvendor-sqlite3.patch
-      - patches/0012-venv-Revert-a-change-from-https-github.com-python-cp.patch
-      - patches/0013-Win-Add-back-the-DLLSuffix-used-to-find-the-openssl-.patch
       # https://github.com/pypa/distutils/pull/53
-      - patches/0014-Use-ranlib-from-env-if-env-variable-is-set.patch
-      - patches/0015-Add-CondaEcosystemModifyDllSearchPath.patch
-      - patches/0016-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
-      - patches/0017-Doing-d1trimfile.patch
+      - patches/0012-Use-ranlib-from-env-if-env-variable-is-set.patch
+      - patches/0013-Add-CondaEcosystemModifyDllSearchPath.patch
+      - patches/0014-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
+      - patches/0015-Doing-d1trimfile.patch
       # https://github.com/python/cpython/pull/23523
-      - patches/0018-cross-compile-darwin.patch
-      - patches/0019-Fix-TZPATH-on-windows.patch
+      - patches/0016-cross-compile-darwin.patch
+      - patches/0017-Fix-TZPATH-on-windows.patch
       # https://github.com/python/cpython/pull/24324
-      - patches/0020-Make-dyld-search-work-with-SYSTEM_VERSION_COMPAT-1.patch
+      - patches/0018-Make-dyld-search-work-with-SYSTEM_VERSION_COMPAT-1.patch
       # https://github.com/pypa/distutils/pull/53
-      - patches/0021-Fix-LDSHARED-when-CC-is-overriden-on-Linux-too.patch
-      - patches/0022-Unvendor-bzip2.patch
-      - patches/0023-Unvendor-libffi.patch
-      - patches/0024-Unvendor-tcltk.patch
-      - patches/0025-unvendor-xz.patch
-      - patches/0026-unvendor-zlib.patch
+      - patches/0019-Fix-LDSHARED-when-CC-is-overriden-on-Linux-too.patch
+      - patches/0020-Unvendor-bzip2.patch
+      - patches/0021-Unvendor-libffi.patch
+      - patches/0022-Unvendor-tcltk.patch
+      - patches/0023-unvendor-xz.patch
+      - patches/0024-unvendor-zlib.patch
+      - patches/0025-Do-not-pass-g-to-GCC-when-not-Py_DEBUG.patch
```

Comment:
We need to add the mailcap patch to this list.

Reply:
Oops, thanks Cheng :)